### PR TITLE
Sync to live

### PR DESCRIPTION
**Pull request by Mikejo5000**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[8b1a66a](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1462/commits/8b1a66aa2bdf306b4866432529faae246fae3278)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Overview-and-Workflow.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/consume-packages/Overview-and-Workflow.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/overview-and-workflow?branch=pr-en-us-1462) |
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1462) |
[docs/create-packages/Creating-a-Package.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/create-packages/Creating-a-Package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/creating-a-package?branch=pr-en-us-1462) |
[docs/guides/Create-UWP-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/guides/Create-UWP-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/guides/create-uwp-packages?branch=pr-en-us-1462) |
[docs/guides/media/UWP-BatchBuild.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/guides/media/UWP-BatchBuild.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/guides/media/uwp-batchbuild.png?branch=pr-en-us-1462) |
[docs/guides/media/UWP-PackageExplorer.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/guides/media/UWP-PackageExplorer.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/guides/media/uwp-packageexplorer.png?branch=pr-en-us-1462) |
[docs/install-nuget-client-tools.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/install-nuget-client-tools.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/install-nuget-client-tools?branch=pr-en-us-1462) |
[docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli?branch=pr-en-us-1462) |
[docs/quickstart/create-and-publish-a-package-using-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/quickstart/create-and-publish-a-package-using-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?branch=pr-en-us-1462) |
[docs/quickstart/install-and-use-a-package-in-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/quickstart/install-and-use-a-package-in-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio?branch=pr-en-us-1462) |
[docs/reference/migrate-packages-config-to-package-reference.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/reference/migrate-packages-config-to-package-reference.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference?branch=pr-en-us-1462) |
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/reference/nuspec.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-1462) |
[docs/what-is-nuget.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/what-is-nuget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/what-is-nuget?branch=pr-en-us-1462) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C6%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201906061638164255-1462%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=qvPIIcfn%2FoYs759Fo%2FattpyaU43RokWzJ8YSrtUNt1U%3D&st=2019-06-06T16%3A34%3A24Z&se=2019-07-07T16%3A39%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Mikejo5000** [comment]
@karann-msft OK to merge master to live?

File: docs/create-packages/Creating-a-Package.md [hunk 1]
```diff
@@ -3,20 +3,26 @@ title: How to create a NuGet package
...
+No matter what your package does or what code it contains, you use one of the NuGet CLI tools, either `nuget.exe` or `dotnet.exe`, to package that functionality into a component that can be shared with and used by any number of other developers. To install NuGet CLI tools, see [Install NuGet client tools](../install-nuget-client-tools.md). Note that Visual Studio does not automatically include a CLI tool.
+
+- For .NET Core and .NET Standard projects that use the SDK-style format ([SDK attribute](/dotnet/core/tools/csproj#additions)), NuGet uses information in the project file directly to create a package. For details, see [Create .NET Standard Packages with Visual Studio 2017](../quickstart/create-and-publish-a-package-using-visual-studio.md) and [NuGet pack and restore as MSBuild targets](../reference/msbuild-targets.md).
+
+- For non-SDK-style projects, follow the steps described in this article to create a package.
```
Comment (zivkan) [inline docs/create-packages/Creating-a-Package.md]: I once asked around internally what we should call non-SDK style projects and "traditional project" was suggested. I like "non-SDK-style project" better, but do we have a standard name for it? NuGet's docs shouldn't be using a different name if dotnet or msbuild docs already have a different name for it.

File: docs/create-packages/Creating-a-Package.md [hunk 2]
```diff
@@ -3,20 +3,26 @@ title: How to create a NuGet package
...
+
+- For projects migrated from `packages.config` to [PackageReference](../consume-packages/package-references-in-project-files.md), use [msbuild pack](../reference/migrate-packages-config-to-package-reference.md#create-a-package-after-migration).
```
Comment (zivkan) [inline docs/create-packages/Creating-a-Package.md]: `msbuild -t:pack` or `dotnet pack`

File: docs/create-packages/Creating-a-Package.md [hunk 3]
```diff
@@ -3,20 +3,26 @@ title: How to create a NuGet package
...
 
 Packaging begins with the compiled code (assemblies), symbols, and/or other files that you want to deliver as a package (see [Overview and workflow](overview-and-workflow.md)). This process is independent from compiling or otherwise generating the files that go into the package, although you can draw from information in a project file to keep the compiled assemblies and packages in sync.
 
 > [!Note]
-> This topic applies to project types other than .NET Core projects using Visual Studio 2017 and NuGet 4.0+. In those .NET Core projects, NuGet uses information in the project file directly. For details, see [Create .NET Standard Packages with Visual Studio 2017](../guides/create-net-standard-packages-vs2017.md) and [NuGet pack and restore as MSBuild targets](../reference/msbuild-targets.md).
+> This topic applies to project types other than .NET Core and .NET Standard projects using Visual Studio 2017 and NuGet 4.0+ (non-SDK-style projects).
```
Comment (zivkan) [inline docs/create-packages/Creating-a-Package.md]: My suggestion:

> This topic applies to non-SDK-style projects

SDK style projects can target the .NET Framework and increasingly others like Xamarin TFMs, so I don't like pretending that SDK style projects are linked only to .NET Core or .NET Standard

File: docs/install-nuget-client-tools.md [hunk 4]
```diff
@@ -59,7 +62,7 @@ Installation:
...
   - The Visual Studio 2017 installer includes the NuGet Package Manager with any workload that employs .NET. To install separately, or to verify that the Package Manager is installed, run the Visual Studio 2017 installer and check the option under **Individual Components > Code tools > NuGet package manager**.
   - The Package Manager UI and Console are unique to Visual Studio on Windows. They are not presently available on Visual Studio for Mac.
-  - Visual Studio does not automatically include the `nuget.exe` CLI, which must be installed separately as described earlier.
+  - Visual Studio does not automatically include a CLI tool, which is required to support NuGet features in the IDE. You can use either the `dotnet` CLI or the the `nuget.exe` CLI. The `dotnet` CLI is installed with some Visual Studio workloads, such as .NET Core. The `nuget.exe` CLI must be installed separately as described earlier.
```
Comment (zivkan) [inline docs/install-nuget-client-tools.md]: The first sentence says that VS doesn't include a CLI tool, but later says that the dotnet cli is installed in some workloads. It seems contradictory.

File: docs/quickstart/create-and-publish-a-package-using-visual-studio.md [hunk 5]
```diff
@@ -18,9 +18,11 @@ It's a simple process to create a NuGet package from a .NET Standard Class Libra
 
 1. Install any edition of Visual Studio 2017 from [visualstudio.com](https://www.visualstudio.com/) with any .NET-related workload. Visual Studio 2017 automatically includes NuGet capabilities when a .NET workload is installed.
 
-1. Install the `nuget.exe` CLI by downloading it from [nuget.org](https://dist.nuget.org/win-x86-commandline/latest/nuget.exe), saving that `.exe` file to a suitable folder, and adding that folder to your PATH environment variable.
+1. Install one of the NuGet CLI tools.
```
Comment (zivkan) [inline docs/quickstart/create-and-publish-a-package-using-visual-studio.md]: The NuGet CLI refers to nuget.exe specifically, so I suggest changing this to "Install of the CLI tools" to keep it generic to mean either nuget.exe or dotnet.exe

File: docs/create-packages/Creating-a-Package.md (hunk 1 above)
Comment (Mikejo5000) [reply docs/create-packages/Creating-a-Package.md]: We have the the "SDK-style project" term published in .NET:
https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/nuget
https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-msbuild
So, it should be OK for now. The other term I've seen is "project SDK".

File: docs/create-packages/Creating-a-Package.md (hunk 3 above)
Comment (Mikejo5000) [reply docs/create-packages/Creating-a-Package.md]: Clarifying...

File: docs/quickstart/create-and-publish-a-package-using-visual-studio.md (hunk 5 above)
Comment (Mikejo5000) [reply docs/quickstart/create-and-publish-a-package-using-visual-studio.md]: Correcting

File: docs/install-nuget-client-tools.md (hunk 4 above)
Comment (Mikejo5000) [reply docs/install-nuget-client-tools.md]: Clarifying

File: docs/create-packages/Creating-a-Package.md (hunk 2 above)
Comment (Mikejo5000) [reply docs/create-packages/Creating-a-Package.md]: Changing to "msbuild -t:pack" I understand that is the recommended method for migrated projects.

File: docs/create-packages/Creating-a-Package.md [hunk 6]
```diff
@@ -3,20 +3,26 @@ title: How to create a NuGet package
...
 ms.author: karann
-ms.date: 12/12/2017
+ms.date: 05/24/2019
 ms.topic: conceptual
 ---
 
 # Creating NuGet packages
 
-No matter what your package does or what code it contains, you use `nuget.exe` to package that functionality into a component that can be shared with and used by any number of other developers. To install `nuget.exe`, see [Install NuGet CLI](../install-nuget-client-tools.md#nugetexe-cli). Note that Visual Studio does not automatically include `nuget.exe`.
+No matter what your package does or what code it contains, you use one of the NuGet CLI tools, either `nuget.exe` or `dotnet.exe`, to package that functionality into a component that can be shared with and used by any number of other developers. To install NuGet CLI tools, see [Install NuGet client tools](../install-nuget-client-tools.md). Note that Visual Studio does not automatically include a CLI tool.
```
[the remark this answers is not in the document]
Comment (Mikejo5000) [reply docs/create-packages/Creating-a-Package.md]: This seems OK to me. "... not automatically" indicates that VS might or might not install the CLI tool.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[7295468](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1462/commits/729546891d13597ec8a2326c8d06e8c84b87b499)_: 

:clock10: **Pending**: waiting for processors ([7](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 7 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by Mikejo5000** [comment]
@zivkan Thanks for your comments! Updating the PR now.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[7295468](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1462/commits/729546891d13597ec8a2326c8d06e8c84b87b499)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Overview-and-Workflow.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/consume-packages/Overview-and-Workflow.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/overview-and-workflow?branch=pr-en-us-1462) |
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1462) |
[docs/create-packages/Creating-a-Package.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/create-packages/Creating-a-Package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/creating-a-package?branch=pr-en-us-1462) |
[docs/guides/Create-UWP-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/guides/Create-UWP-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/guides/create-uwp-packages?branch=pr-en-us-1462) |
[docs/guides/media/UWP-BatchBuild.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/guides/media/UWP-BatchBuild.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/guides/media/uwp-batchbuild.png?branch=pr-en-us-1462) |
[docs/guides/media/UWP-PackageExplorer.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/guides/media/UWP-PackageExplorer.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/guides/media/uwp-packageexplorer.png?branch=pr-en-us-1462) |
[docs/install-nuget-client-tools.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/install-nuget-client-tools.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/install-nuget-client-tools?branch=pr-en-us-1462) |
[docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli?branch=pr-en-us-1462) |
[docs/quickstart/create-and-publish-a-package-using-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/quickstart/create-and-publish-a-package-using-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?branch=pr-en-us-1462) |
[docs/quickstart/install-and-use-a-package-in-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/quickstart/install-and-use-a-package-in-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio?branch=pr-en-us-1462) |
[docs/reference/migrate-packages-config-to-package-reference.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/reference/migrate-packages-config-to-package-reference.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference?branch=pr-en-us-1462) |
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/reference/nuspec.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-1462) |
[docs/what-is-nuget.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/what-is-nuget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/what-is-nuget?branch=pr-en-us-1462) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C7%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201906071640571199-1462%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=b53ll2Z2XmC4uRU%2B7lrlB34Am9WEJK6dtlBHAghcHM4%3D&st=2019-06-07T16%3A36%3A54Z&se=2019-07-08T16%3A41%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**